content/en/docs/concepts/storage/volumes.md

```diff
@@ -678,6 +678,10 @@ spec:
       type: FileOrCreate
 ```
 
+{{< note >}}
+Volumes' names must be different. For example: ```mydir``` & ```myfile``` as shown in the above [`FileOrCreate` configuration](#hostpath-fileorcreate-example)).
```

Comment:
Just a suggestion i think it's better we write as `volume's`

Reply:
Thanks for your suggestion @Kartik494, but I think `volume's` will be singular possessive, whereas `volumes'` is plural possessive